README.md

```diff
@@ -1,13 +1,14 @@
 # OpenRCT2/OpenGraphics
 
-**OpenGraphics** is a project established to replace the base graphics of Rollercoaster Tycoon 2. The aim is to create a set of freely-useable graphics that can be used with [OpenRCT2](https://github.com/OpenRCT2/OpenRCT2), to decouple its reliance on the copyrighted material from the original game and allow anyone to play the game for free.
+**OpenGraphics** is a project established to replace the base graphics of Rollercoaster Tycoon 2. The aim is to create a set of freely-useable graphics that can be used with [OpenRCT2](https://github.com/OpenRCT2/OpenRCT2), in order to decouple its reliance on the copyrighted material from the original game, and to allow anyone to play the game for free.
 
 ## Getting Started
 
-We're in the process of setting up pipelines and deciding on the direction of OpenGraphics. If you have experience with art or tools development than consider checking out our discussions about establishing the rendering process:
+We're in the process of setting up pipelines and deciding on the direction of OpenGraphics... If you have experience with art or tool development then consider checking out our discussions about establishing the rendering process:
```

Comment:
The elipsis shouldn't be used in descriptions.

Reply:
I just changed the issues.